next_generation/pint_server/app.py

```diff
@@ -225,12 +231,14 @@ def _get_azure_images_for_region_state(region, state):
 
 
 def get_provider_images_for_region_and_state(provider, region, state):
+    images = []
     if provider == 'microsoft':
         return _get_azure_images_for_region_state(region, state)
 
-    images = PROVIDER_IMAGES_MODEL_MAP[provider].query.filter(
-        PROVIDER_IMAGES_MODEL_MAP[provider].region == region,
-        PROVIDER_IMAGES_MODEL_MAP[provider].state == state)
+    if hasattr(PROVIDER_IMAGES_MODEL_MAP[provider], 'region') and hasattr(PROVIDER_IMAGES_MODEL_MAP[provider], 'state'):
```

Comment:
I think pep8 will complain about the line length here...

Reply:
Fixed